assets/src/scss/blocks.scss

```diff
@@ -9,7 +9,7 @@
 @import "blocks/Cookies/CookiesStyle";
 @import "blocks/Counter/CounterStyle";
 @import "blocks/ENForm/ENFormStyle";
-@import "blocks/Happypoint/HappypointStyle";
+@import "blocks/HappyPoint/HappypointStyle";
```

Comment:
The style file's case should also be fixed: HappypointStyle => HappyPointStyle

Reply:
Fixed!